CHANGELOG.md

```diff
@@ -4,6 +4,10 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog 1.0.0].
 
+## [Release 13.0.1]
```

Comment:
When updating the CHANGELOG you also need to update the links down at the very bottom of the document, it's not (yet) something which happens by magic.